app/models/event.rb

```diff
@@ -9,23 +9,30 @@ class Event < ApplicationRecord
   validate :no_collision
 
   def start_time_must_be_lower_than_end_time
-    errors.add(:start_time, "Can't be higher than end_time") if start_time.try(:>=, end_time)
+    return unless start_time && end_time
```

Comment:
Do you have to check for `start_time` and `end_time` when you already validate their presence?